File: js/service/test1.js
import * as skynet from "skynet";
let amount_pre_sec = 0;
let total = 0;
let begin_ts = 0;
let ts = 0;
skynet.start(() => {
    skynet.dispatch("lua", () => {
        amount_pre_sec++;
        total++;
        let now = skynet.now();
        if (ts == 0) {
            ts = now;
            begin_ts = now;
        }
        if (now - ts > 100) {
            console.log("js msg per sec:", amount_pre_sec, Math.floor(total / Math.floor((now - begin_ts) / 100)));
            amount_pre_sec = 0;
            ts = now;
        }
    });
    skynet.register(".testjs");
});

File: service/test1.lua
local skynet = require "skynet"
require "skynet.manager"

local amount_pre_sec = 0;
local total = 0;
local begin_ts = 0;
local ts = 0;
skynet.start(function()
    skynet.dispatch("lua", function(session, source, ...)
        local now = skynet.now()
        amount_pre_sec = amount_pre_sec + 1
        total = total + 1

        if ts == 0 then
            ts = now
            begin_ts = now
        end
        
        if now - ts > 100 then
            print("lua msg per sec:", amount_pre_sec, total // ((now - begin_ts)//100))
            amount_pre_sec = 0
            ts = now
        end
    end)
    skynet.register(".testlua")
end)